rhscl.yaml

```diff
@@ -74,6 +101,33 @@ templates:
     envs:
       - "BASH_ENV        /etc/profile.d/cont-env.sh"
     cmd: ["container-usage"]
+    devel_libs: |
+      \
+      autoconf \
+      automake \
+      bsdtar \
+      curl-devel \
+      epel-release \
+      gcc-c++ \
+      gdb \
+      gettext \
+      libxml2-devel \
+      libxslt-devel \
+      lsof \
+      make \
+      mysql-devel \
+      mysql-libs \
+      openssl-devel \
+      postgresql-devel \
+      procps-ng \
+      scl-utils \
+      sqlite-devel \
+      tar \
+      unzip \
+      wget \
+      which \
+      yum-utils \
+      zlib-devel \
```

Comment:
@rhcarvalho that would mean 1.5MB more, without docs less probably, but still, do we have a good reason to have it there? we should be careful with adding packages since images would grow fast.
@sYnfo @notroj @humaton @strzibny -- anyone against findutils?